tests/unittests/test_implicit.py

```diff
@@ -61,20 +65,21 @@ def dftb_calculator(device, skf_file: str):
     overlap_feed = SkFeed.from_database(skf_file, species, 'overlap', device=device)
     occupation_feed = SkfOccupationFeed.from_database(skf_file, species, device=device)
     hubbard_feed = HubbardFeed.from_database(skf_file, species, device=device)
-    repulsive_feed = RepulsiveSplineFeed.from_database(skf_file, species, device=device)
+    repulsive_feed = PairwiseRepulsiveEnergyFeed.from_database(skf_file, species, device=device)
 
     # set up the calculator
     return Dftb2(hamiltonian_feed, overlap_feed, occupation_feed,
                  hubbard_feed, r_feed=repulsive_feed, filling_scheme=None)
 
 
 def implicit_gradient_helper(mol: Geometry, orbs: OrbitalInfo, dftb_calculator, device):
-
+    dftb_calculator.grad_mode = "direct"
     energy_direct = dftb_calculator(mol, orbs, grad_mode='direct')
```

Comment:
energy_direct = dftb_calculator(mol, orbs, ~~grad_mode='direct'~~)
Hey Adam, as `grad_mode` now is an argument of the calculator, we may not need to specify it during the forward calling.